packages/suite/src/components/suite/WordInput/index.tsx

```diff
@@ -36,20 +36,42 @@ const SelectWrapper = styled.div`
     @media only screen and (max-width: ${variables.SCREEN_SIZE.MD}) {
         width: 280px;
     }
+`;
+
+const StyledList = styled(List)`
+    box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2);
+    padding: 8px;
+    position: static !important;
 
-    .react-select__menu {
-        padding-left: 8px;
+    & > div {
```

Comment:
FYI the`&` symbol not necessary at all in StyledComponents in those cases :)

Reply:
Thanks, 610d0d501c4a3ae2df8c5126571d5b1ee85984bd